contrib/test_jenkins.sh

```diff
@@ -958,10 +958,13 @@ test_jucx() {
                         fi
                         echo "Running standalone benchamrk on $iface"
 
-                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar org.ucx.jucx.examples.UcxReadBWBenchmarkReceiver s=$server_ip p=$JUCX_TEST_PORT 2>&1 > bindings/java/src/main/native/build-java/jucx_bench.log & 
+                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar \
+				 org.ucx.jucx.examples.UcxReadBWBenchmarkReceiver \
+				 s=$server_ip p=$JUCX_TEST_PORT &
                         sleep 10
-                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar  org.ucx.jucx.examples.UcxReadBWBenchmarkSender s=$server_ip p=$JUCX_TEST_PORT t=10000000
-                        cat bindings/java/src/main/native/build-java/jucx_bench.log
+                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar  \
+				 org.ucx.jucx.examples.UcxReadBWBenchmarkSender \
+				 s=$server_ip p=$JUCX_TEST_PORT t=10000000
```

Comment:
forgot to mention last time - need to add "wait" shell command after, to wait until server exits and capture its return code